src/commonComponents/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,32 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//CSS
+import './Checkbox.css';
+function Checkbox(props) {
+  const { className, isChecked, label, htmlFor } = props;
+
+  return (
+    <div className={`${className} checkbox`}>
+      <input type='checkbox' id={htmlFor} value={isChecked} />
+      <p>{label}</p>
```

Comment:
What's the point of htmlFor, if you're not using it anywhere?

Reply:
htmlFor is used in case of checkbox, so that when you clock on the label checkbox should change it's state, you might have to wire it accordingly